### ISSUE TITLE
html

### ISSUE THREAD

**Issue by devamirsadeghian** [devamirsadeghian/final#1]
<bdo dir=""></bdo>
<body dir="rtl"> 

</body>

<footer>  </footer>
<header>  </header>

<h1></h1>
<h2></h2>
<h3></h3>
<h4></h4>
<h5></h5>
<h5></h5>
<h6></h6>

<hr>

<p></p>
<br>
<pre></pre>

<b></b>
<strong></strong>  

<i></i>
<em></em>          

<small></small>
<sup></sup>
<sub></sub>
<del></del>

<q></q>
<blockquote cite=""></blockquote>

<abbr title="خلاصه">محتوی نمایشی</abbr>
<address></address>

<a href="" target="_self""_blank"></a>
<img src="" alt="نوشته که به جای عکسی که بارگزاری شده">

<img src="" alt="نوشته که به جای عکسی که بارگزاری شده" usemap="#id">
<map name="id"></map>
<area shape="" coords="" href="">       
<area shape="" coords="" href="">

<i class="fas-------------------"></i>    

<iframe src="" width="" height=""></iframe>

<div></div>
<span></span>
<button></button>
<nav></nav>

<label for=""></label>

<table>
    <tr>
        <td colspan="2">
            
        </td>
        <td rowspan="2">

        </td>
    </tr>
    <tr>
        <td>

        </td>
        <td>

        </td>
    </tr>
</table>

<ol>              
    <li></li>
    <li></li>
    <li></li>
</ol>

<ul>              
    <li></li>
    <li></li>
    <li></li>
</ul>

<body dir="rtl">
    <fieldset>
        <legend>سرتیتر فرم</legend>

        <form action="">
            @csrf   
            <table>
                <tr>
                    <td>
                        متن:
                    </td>
                    <td>
                        <input type="text" size="عدد" placeholder="متن" maxlength="عدد">
                    </td>
                </tr>

                <tr>
                    <td>
                        رمز:
                    </td>
                    <td>
                        <input type="password" size="عدد" placeholder="متن" maxlength="عدد">
                    </td>
                </tr>

                <tr>
                    <td>
                        ایمیل:
                    </td>
                    <td>
                        <input type="email" size="عدد" placeholder="متن">
                    </td>
                </tr>

                <tr>
                    <td>
                        گزینه رادیویی:
                    </td>
                    <td>
                        <input type="radio" name="یکسان">محتوا 1
                        <input type="radio" name="یکسان">محتوا 2
                    </td>
                </tr>

                <tr>
                    <td>
                        چک باکسی:
                    </td>
                    <td>
                        <input type="checkbox" checked>محتوا 1
                        <input type="checkbox">محتوا 2
                        <input type="checkbox">محتوا 3
                    </td>
                </tr>

                <tr>
                    <td>
                        دریافت فایل:
                    </td>
                    <td>
                        <input type="file">
                    </td>
                </tr>

                <tr>
                    <td>
                        متن بزرگ:
                    </td>
                    <td>
                        <textarea cols="30" rows="10"></textarea>
                    </td>
                </tr>

                <tr>
                    <td>
                        عنوان:
                    </td>
                    <td>
                        <select>
                            <option selected>محتوا</option>
                            <option>محتوا</option>
                            <option>محتوا</option>
                        </select>
                    </td>
                </tr>

                <tr>
                    <td>
                        عنوان:
                    </td>
                    <td>
                        <select>
                            <optgroup label="گروه1">
                                <option selected>محتوا</option>
                                <option>محتوا</option>
                                <option>محتوا</option>
                            </optgroup>
                            <optgroup label="گروه2">
                                <option>محتوا</option>
                                <option>محتوا</option>
                                <option>محتوا</option>
                            </optgroup>
                        </select>
                    </td>
                </tr>

                <tr>
                    <td>
                        <input type="submit" value="محتوا">   
                        <input type="reset" value="محتوا">    
                    </td>
                </tr>

            </table>

        </form>
        
    </fieldset>
</body>

selector#id*number>{value}

p#iddd*3>{value}
<p id="iddd">value</p>
<p id="iddd">value</p>
<p id="iddd">value</p>

selector.class*number>{value}

p#class_tttt*3>{value}
<p class="class_tttt">value</p>
<p class="class_tttt">value</p>
<p class="class_tttt">value</p>

<!DOCTYPE html>
<head>
</head>
<body>
    <form  action=" <?php echo $_SERVER[PHP_SELF]" ?> method="get or post"></form>
    <input type="text" name="txt_1">
</body>
</html>

<script src="jquery-3.7.1.min.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
